docs/UserGuide.adoc

```diff
@@ -412,23 +412,30 @@ Selects the 2nd guest in the displayed list.
 `select 100` +
 Selects the 100th room in the displayed list.
 
+// tag::service[]
 === Provide Room Service: `service` image:icon-key.png[width="32"]
 
 Charges a room service to a room  +
 Format: `service r/ROOM_NUMBER no/ITEM_NUMBER [c/COST]`
 
 ****
 * ITEM_NUMBER refers to the number in the menu given to each type of service offered by the hotel.
-* This could be room service, on-demand video streaming, bath supplies, mini bar, etc.
 * The cost can be specified as well, if the guest is to be charged an amount that is
   different from the cost in the menu. Note that the cost has to follow a strict format
   such as 100.00, i.e. with two decimal places. The dollar part of the cost should also
   not exceed `Integer.MAX_VALUE`.
 * Any expenses added with this command will be accounted for under the Expenses portion
-  of the respective rooms.
+  of the respective rooms. The Expenses of the room will also be displayed on the
+  right panel.
 * Only occupied rooms (i.e. rooms with checked-in guests) can have expenses charged to it.
+* It is possible to charge negative values as well. This can be used in cases such as when
```

Comment:
Consider using the `TIP` or `IMPORTANT` icon as specified in the Terminology section